src/Altinn.App.Api/Models/InstanceResponse.cs

```diff
@@ -161,5 +172,51 @@ public class InstanceOwnerResponse
     /// <summary>
     /// Party information for the instance owner.
     /// </summary>
-    public required Party Party { get; init; }
+    public required PartyResponse Party { get; init; }
+}
+
+/// <summary>
+/// Class representing a party
+/// </summary>
+public sealed class PartyResponse
+{
+    /// <summary>
+    /// Gets or sets the ID of the party
+    /// </summary>
+    public required int PartyId { get; init; }
+
+    /// <summary>
+    /// Gets or sets the UUID of the party
+    /// </summary>
+    public required Guid? PartyUuid { get; init; }
+
+    /// <summary>
+    /// Gets or sets the type of party
+    /// </summary>
+    public required PartyType PartyTypeName { get; init; }
+
+    /// <summary>
+    /// Gets the parties ssn
+    /// </summary>
+    public required string SSN { get; init; }
```

Comment:
This should probably not be required and not nullable?

Reply:
Nullability is disabled in this file. Required is just to make the mapping code less error-prone